src/TLuaInterpreterUI.cpp

```diff
@@ -1463,8 +1464,14 @@ int TLuaInterpreter::getWindowWrap(lua_State* L)
 // Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#hasFocus
 int TLuaInterpreter::hasFocus(lua_State* L)
 {
-    const Host& host = getHostFromLua(L);
-    lua_pushboolean(L, host.mpConsole->hasFocus()); //FIXME
+    QString windowName;
+    if (lua_gettop(L) > 0) {
```

Comment:
It can't be negative can it? So this is simpler:
```suggestion
    if (lua_gettop(L)) {
```